src/xdocs/config_imports.xml

```diff
@@ -1726,21 +1802,52 @@ import java.util.stream.IntStream;
   &lt;property name="message" value="^'java\..*'.*"/&gt;
 &lt;/module&gt;
         </source>
+        <p>
+          Example:
+        </p>
+        <source>
+import static javax.swing.WindowConstants.*; // Violation; all static imports comes at bottom
 
-        <p>To configure the check so that it matches default NetBeans formatter configuration
-           (tested on v8):</p>
+import java.net.URL; // Violation; Wrong order for 'java.net.URL'
+import java.security.KeyManagementException;
+import javax.net.ssl.TrustManager; // OK; no blank line between "javax" and "java".
+
+import static java.awt.Button.A; // Violation;  Extra separation in import group
+import static java.lang.Math.PI; // OK
+import static java.lang.Math.abs; // OK, alphabetical case sensitive ASCII order, 'P' &lt; 'a'
+
+public class SomeClass { }
+        </source>
+        <p>
+          To configure the check so that it matches default NetBeans formatter configuration
+          (tested on v8):
+        </p>
         <ul>
           <li>groups of type imports are not defined, all imports will be sorted as a one
               group</li>
           <li>static imports are not separated, they will be sorted along with other imports</li>
         </ul>
-
         <source>
 &lt;module name=&quot;ImportOrder&quot;&gt;
   &lt;property name=&quot;option&quot; value=&quot;inflow&quot;/&gt;
 &lt;/module&gt;
         </source>
+        <p>
+          Example:
+        </p>
+        <source>
+import static java.io.File.createTempFile;
+
+import static java.lang.Math.PI; // Violation; Static imports are not separated
+import java.lang.Math.sqrt;
+import javax.swing.JComponent; // Ok; Imports are grouped as one
```

Comment:
// Ok or // ok ? We should try to be consistent across examples :)

Reply:
done 
i actually already did this before just waiting for your reply.

Reply:
Please do not reply "done" until your changes are pushed.  This goes for all comments below this.

Reply:
Resolved.

Reply:
Okay